### PR TITLE
corrections to 3.1 blog post

### PR DESCRIPTION
**Pull request by MelSumner**
- [x] update Module Unification to include "new file layout" to more clearly set expectations about terminology
- [x] include information about named arguments

### CODE REVIEW

File: source/blog/2018-04-13-ember-3-1-released.md [hunk 1]
```diff
@@ -200,7 +210,7 @@ In Ember 3.1 we've corrected the implementation to act like a proper closure. In
 For more information about this change see
 [emberjs/ember.js#15287](https://github.com/emberjs/ember.js/pull/15287).
 
-### Deprecations in Ember 3.1
+### Deprecations in Ember 3.1 (2)
```
Comment (locks) [inline source/blog/2018-04-13-ember-3-1-released.md]: this is confusing for me :/

File: source/blog/2018-04-13-ember-3-1-released.md [hunk 2]
```diff
@@ -440,9 +462,9 @@ See what I mean? Totally awesome. <3
...
+**Module Unification (new file layout) Continues** - You can now generate an addon using the Module Unification layout [(#7490)](https://github.com/ember-cli/ember-cli/pull/7490)! Use the command `MODULE_UNIFICATION=true ember addon my-addon` to try it out [(#7658)](https://github.com/ember-cli/ember-cli/pull/7658). We also improved the logic to support addons that use Module Unification [(#7660)](https://github.com/ember-cli/ember-cli/pull/7660), added the blueprint for a dummy app to addons that use Module Unification [(#7667)](https://github.com/ember-cli/ember-cli/pull/7667), and updated the version of Ember used in Module Unification [(#7678)](https://github.com/ember-cli/ember-cli/pull/7678).
```
Comment (locks) [inline source/blog/2018-04-13-ember-3-1-released.md]: sub-headers like above?

File: source/blog/2018-04-13-ember-3-1-released.md (hunk 1 above)
Comment (MelSumner) [reply source/blog/2018-04-13-ember-3-1-released.md]: ok i'll just remove it for now. I have a template worked out for future posts so we get the consistency we're after.

File: source/blog/2018-04-13-ember-3-1-released.md (hunk 2 above)
Comment (MelSumner) [reply source/blog/2018-04-13-ember-3-1-released.md]: not on this one. just want to ship it.

**Pull request merged**